users/config.rst

```diff
@@ -523,7 +576,7 @@ address
         The host name will be used on the given port and connections will be
         attempted via both IPv4 and IPv6, depending on name resolution.
 
-    ``dynamic``
+    dynamic
```

Comment:
I think having this formatted as code makes sense.  In fact, that should be applied to all option names IMO.